### PR TITLE
Replica improvements

### PR DESCRIPTION
**Pull request by hannahbrucemacdonald**

### CODE REVIEW

**Comment by hannahbrucemacdonald** [comment]
Closes #661

File: perses/annihilation/lambda_protocol.py [hunk 1]
```diff
@@ -102,6 +102,12 @@ def __init__(self, functions='default'):
                                   lambda x: x,
                                   'lambda_torsions':
                                   lambda x: x}
+            elif self.type == 'ele-scaled':
```
Comment (jchodera) [inline perses/annihilation/lambda_protocol.py:105]: Maybe add a comment here that this is "linear scaling of electrostatic energy for insertion and deletion"?

File: perses/app/relative_setup.py [hunk 2]
```diff
@@ -263,9 +262,9 @@ def __init__(self, ligand_input, old_ligand_index, new_ligand_index, forcefield_
 
         # Create SystemGenerator
         from openmmforcefields.generators import SystemGenerator
-        forcefield_kwargs = {'removeCMMotion': False, 'ewaldErrorTolerance': self._pme_tol, 'nonbondedMethod': self._nonbonded_method,'constraints' : app.HBonds, 'hydrogenMass' : self._hmass}
```
Comment (dominicrufa) [inline perses/app/relative_setup.py:266]: i missed something; why is te nonbonded method not being specified anymore?

File: perses/annihilation/lambda_protocol.py (hunk 1 above)
Comment (hannahbrucemacdonald) [reply perses/annihilation/lambda_protocol.py:122]: Done in docstrings

File: perses/app/relative_setup.py (hunk 2 above)
Comment (hannahbrucemacdonald) [reply perses/app/relative_setup.py:266]: Good spot, this is to do with openmm-forcefields, where whatever nonbonded method we chose was being overwritten by the default for `periodic_forcefield_kwargs` (which is PME anyway, but the reason that RF wasn't working before)

openmm-forcefield fix: https://github.com/openmm/openmmforcefields/pull/109
perses issue: https://github.com/choderalab/perses/issues/661

**Comment by hannahbrucemacdonald** [comment]
closes #661

File: perses/app/relative_setup.py (hunk 2 above)
Comment (dominicrufa) [reply perses/app/relative_setup.py:266]: so if i wanted `NoCutoff` for vacuum, what would i do?

**Pull request merged**